test/helpers/projects_helper_test.rb

```diff
@@ -146,7 +146,7 @@ class ProjectsHelperTest < ActionView::TestCase
     it 'should return project managers list' do
       project_manager = create(:manage)
       @project = project_manager.target
-      _(project_managers_list).must_match project_manager.account.name
+      _(project_managers_list).to_h[:target].match(/(a href|\/\/).*/)
```

Comment:
Can I get an explanation for the change from a specific account name to a regex on an href?

Reply:
Previously we were using the disabled_button code from buttons_helper , which was not using the link for redirection.

So, this test case started failing because instead of name , Now its coming link in the account name.

So , I modified the test case according to that.